### PR TITLE
Increase the scallbar width

### PR DESCRIPTION
**Pull request by mrburns-42**

### CODE REVIEW

**Comment by skronenwerth** [comment]
#6275

**Comment by rullzer** [comment]
@nextcloud/designers

**Comment by skronenwerth** [comment]
Hi, this is my first pull request. For my understanding, what does not work to prevent errors in the automatic checks?

**Comment by rullzer** [comment]
@skronenwerth you can't help the swift errors (they happen on some CI hosts we are looking into it).

We do require your to sign off on your commits (see https://github.com/nextcloud/server/blob/master/contribute/developer-certificate-of-origin)

To do so for this simply run:

```sh
git commit --amend -s
git commit push -f
```

**Comment by skronenwerth** [comment]
@rullzer Thanks for the information, at the next commit (which I hope will be) I will do it that way.

**Comment by juliushaertl** [comment]
@nextcloud/designers How about using the browser default width for the main content scrollbar.
For things like the app navigation sidebar we could also consider something like only showing the scroll bar on hover: https://codepen.io/kqlambert/pen/tbJCD

**Comment by skjnldsv** [comment]
> only showing the scroll bar on hover: https://codepen.io/kqlambert/pen/tbJCD

U like it!

**Comment by skronenwerth** [comment]
Excuse my attempts to commit. In the past I have worked with svn and rarely with github.

**Comment by jancborchardt** [comment]
@skronenwerth welcome to Nextcloud design :) For design-related changes, a before/after screenshot is always great, and some reasoning for the change. Thank you :)

**Comment by jancborchardt** [comment]
Alsp, as mentioned in the original issue, can you make it so the scrollbar width only changes on hover?

Also @juliushaertl's suggestion of only showing it on mouse move at all (like on macOS) is good too – let's work toward that. :)

File: core/css/styles.scss [hunk 1]
```diff
@@ -161,8 +161,13 @@ body {
 /* SCROLLING */
 
 ::-webkit-scrollbar {
+<<<<<<< HEAD
+	width: 0.7em;
+	height: 0.7em;
+=======
 	width: 8px;
 	height: 8px;
+>>>>>>> 9b048a51d53c22b97723d548d9a1fb6a4e8ae63c
```
Comment (MorrisJobke) [inline core/css/styles.scss]: Looks like a wrong conflict resolution.

Comment (skronenwerth) [reply core/css/styles.scss]: Ja, das war es. Es war ein Versehen das ich das pushed hatte.

**Comment by skronenwerth** [comment]
> Alsp, as mentioned in the original issue, can you make it so the scrollbar width only changes on hover?

I will try it.

**Comment by MorrisJobke** [comment]
As there was no activity for 2 months I will close this PR. Don't worry your work is not lost. Just create a new PR with the updated functionality or reopen this one here.

Thanks for your contribution anyways. 👍

**Pull request closed without merging**